ansible/playbooks/roles/upgrade/tasks/opensearch/migrate-kibana.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: Kibana migration | Load deafults from Opensearch Dashboards role
+  include_vars:
+    file: roles/opensearch_dashboards/defaults/main.yml
+
+- name: Kibana migration | Load vars from Opensearch Dashboards role  # requires epicli upgrade -f <manifest>.yml
+  include_vars:
+    file: roles/opensearch_dashboards/vars/main.yml
+
+- name: Kibana migration | Stop Kibana service
+  systemd:
+    name: kibana
+    enabled: no
+    state: stopped
+
+- name: Kibana migration | Download Opensearch Dashboards binary
+  include_role:
+    name: download
+    tasks_from: download_file
+  vars:
+    file_name: "{{ file_name_version.opensearch_dashboards[ansible_architecture] }}"
+
+- name: Kibana migration | Create opensearch-dashboards user
+  user:
+    name: "{{ specification.dashboards_os_user }}"
+    password: "{{ specification.dashboards_password }}"
+    state: present
+    shell: /bin/bash
+
+- name: Kibana migration | Create OPSD directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ specification.dashboards_os_user }}"
+    group: "{{ specification.dashboards_os_user }}"
```

Comment:
Maybe we can specify use of group different than name?

Reply:
As discussed before: there are two approaches and it's hard to say which is better.
Probably would be good to discuss it within the team before changing.